gbfs/views.py

```diff
@@ -46,7 +48,18 @@ def gbfs(request):
 
 def gbfsSystemInformation(request):
     if request.method == "GET":
-        data = {"system_id": "TBD", "language": "TBD", "name": "TBD", "timezone": "TBD"}
+        bsp = preferences.BikeSharePreferences
+        lang = translation.get_language()
+        twoletterlangcode = lang.split("-")[0]
```

Comment:
personally, I wouldn't split the langcode. the gbfs spec uses IETF BCP 47 there, `language-region/variant` is defined there. it only has to be consistent in all our usage.

Reply:
fixed in  abfcd08